src/zzz_od/application/driver_disc_read/driver_disc_read_app.py

```
@@ -0,0 +1,625 @@
+                    images = [img for _, img in items]
+
+                    # 批量OCR识别
+                    try:
+                        # 直接调用底层text_recognizer进行批量推理
+                        # 因为图片已经预裁剪到精确区域，不需要文本检测
+                        rec_results = worker_ocr._model.text_recognizer(images)
+                        
+                        for idx, (text, score) in zip(indices, rec_results):
+                            all_results[idx][area_name] = text.strip()
+                    except Exception as e:
+                        # 降级：批量调用失败时逐个处理
```

Ruff (0.14.0)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)